include/plat/spike/plat/machine/hardware.h

```diff
@@ -31,10 +31,15 @@
 #define PADDR_BASE 0x0lu
 #endif
 
+#ifndef CONFIG_BUILD_ROCKET_CHIP
 /* This represents the physical address that the kernel image will be linked to. This needs to
  * be on a 1gb boundary as we currently require being able to creating a mapping to this address
  * as the largest frame size */
 #define PADDR_LOAD 0xC0000000lu
+#else
+/* The Rocket-Chip hardware only has 256MiB of Memory. */
```

Comment:
The ```Rocket-Chip``` itself is a parameterised i.e. it's not restricted/bound to 256MiB of memory. For example, I am running the Rocket Chip on a board that has 1 GiB of memory. I'd suggest you rename the platform to your specific board that runs the Rocket Chip (e.g. ```PLAT_ROCKET_ZEDBOARD```). Otherwise, if you want to keep the config name as is, the Rocket Chip parameters should be also seL4 parameters to allow your code to run on other Rochet-Chip-based board.

[ditto](https://github.com/seL4/seL4/pull/94/files#r204326181)

Reply:
will rename BUILD_ROCKET_CHIP to BUILD_ROCKET_CHIP_ZEDBOARD